sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -81,21 +87,30 @@ export class AssessmentControllerService {
       offeringStudyEndDate: getDateOnlyFormat(assessment.offering.studyEndDate),
       disbursement: this.populateDisbursementAwardValues(
         assessment.disbursementSchedules,
+        { maskMSFAA: options?.maskMSFAA },
       ),
     };
   }
 
   /**
    * Disbursement data is populated with dynamic key in a defined pattern to be compatible with form table.
    * @param disbursementSchedules disbursement schedule details.
-   * @param includeDocumentNumber when true document number is mapped
+   * @param options for NOA.
+   * - `includeDocumentNumber` when true document number is mapped
    * to disbursement dynamic data.
+   * - `maskMSFAA` mask MSFAA or not.
    * @returns disbursement dynamic award data.
    */
   private populateDisbursementAwardValues(
     disbursementSchedules: DisbursementSchedule[],
-    includeDocumentNumber = false,
+    options?: {
+      includeDocumentNumber?: boolean;
+      maskMSFAA?: boolean;
+    },
   ): Record<string, string | number> {
+    // Setting default value.
+    const includeDocumentNumber = options?.includeDocumentNumber ?? false;
+    const maskMSFAA = options?.maskMSFAA ? options.maskMSFAA : false;
```

Comment:
Shouldnt the maskMSFAA default value should be true?
const maskMSFAA = options?.maskMSFAA ? !options.maskMSFAA : true; so in a situation the options maskMSFAA is not given then it should not be shown. Thoughts?? @andrewsignori-aot @ann-aot @dheepak-aot @sh16011993 @andrepestana-aot

Reply:
I would be ok changing it to `true` just for security reasons. Either way, an E2E test can also check for it.